### DISCUSSION TITLE
Stuff Requests

### DISCUSSION THREAD

**Discussion by bonbonboi** [typefully/minimal-twitter#138] [Ideas]
These basic stuff needs to be added: Hide Retweets with whitelist | Auto Refresh Timer with auto scroll to top | Enable Show Trends under Search bar | Hide the top Tweet Composer box | Navigation Labels doesn't work as I expected (To always show labels on the right of the icons).

**Comment by ghost** [comment]
- Trends: https://github.com/typefully/minimal-twitter/pull/141/commits/ab9081af92e06b83af8bfdce0cf30dfe067b45d7
- Navigation labels should be fixed!
